### DISCUSSION TITLE
How to manage local data in Electron app?

### DISCUSSION THREAD

**Discussion by luxuereal** [luxuereal/budget_nextron_sqlite#15] [Ideas]
Hello, everyone!
Let us discuss about how to manage local data in Electron app.
I want good answers from you.
Thank you.

**Comment by pearlights** [comment]
Yeah, I think we can use SQLite in electron app. How about?

**Reply by luxuereal** [reply]
Yeah, I also know SQLite, but electron app have to work in case of both of offline and online.

**Reply by pearlights** [reply]
Oh, idk. I need to inspect about that.

**Comment by vik-splav** [comment]
> Hello, everyone! Let us discuss about how to manage local data in Electron app. I want good answers from you. Thank you.

Hello, there is new technology. NxDB
